t/plugin/error-log-logger.t

```diff
@@ -281,7 +287,115 @@ passed
 
 
 
-=== TEST 9: want to reload the plugin by route
+=== TEST 9: log a warn level message (schema compatibility testing)
```

Comment:
Better to add test at the end of file

Reply:
we have resetting environment tasks at the end.